test/integration/dialects/postgres/connection-manager.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+/* jshint -W030 */
+var chai = require('chai')
+  , expect = chai.expect
+  , Support = require(__dirname + '/../../support')
+  , dialect = Support.getTestDialect()
+  , DataTypes = require(__dirname + '/../../../../lib/data-types')
+  , _ = require('lodash');
+
+if (dialect.match(/^postgres/)) {
```

Comment:
Also I think its used here to check for `postgres` and `postgres-native` both. @mickhansen